portfolio/src/main/webapp/style.css

```diff
@@ -67,4 +63,20 @@ td, tr {
 
 th {
     padding: 10px;
+}
+
+#gallery {
+  margin-left: 300px;
+  margin-right: 150px;
+}
+
+#gallery img {
+  height: 500px;
+}
+
+img:hover {
```

Comment:
nit: I'm not sure if the hover opacity is necessary, esp. because it seems to imply that the images are clickable.

Reply:
That's fair. I'm thinking about making it so a caption shows up when you hover.

Reply:
Just kidding, turns out that is SUPER difficult to do.

Reply:
No worries! Just a suggestion :)

Reply:
I did manage to figure out something kind of similar with JS in the end. I'll make a pull request for it after this gets approved!